backend/flaskr/model_utils.py

```diff
@@ -0,0 +1,69 @@
+"""Utility functions to be used for model creation, training, and utilization"""
+
+from backend.flaskr.yelp_api_utils import YelpAPI
+
+IMPORTANCE_KEYS = ["food", "service", "atmosphere", "value"]
+
+def get_categories(term):
+    """
+    Semantically parse a raw search term into Yelp categories
+    :param term: search term the user entered to be semantically parsed
+    :return: List of Yelp catefories that the search term best matches
+    """
+    # Query NYC with the search term in a 15 mile radius
```

Comment:
20 mile radius?

Reply:
Whoops changed the code but not the comment.